colors/onedark.vim

```diff
@@ -259,6 +259,9 @@ call s:h("Visual", { "fg": s:visual_black, "bg": s:visual_grey }) " Visual mode
 call s:h("VisualNOS", { "bg": s:visual_grey }) " Visual mode selection when vim is "Not Owning the Selection". Only X11 Gui's gui-x11 and xterm-clipboard supports this.
 call s:h("WarningMsg", { "fg": s:yellow }) " warning messages
 call s:h("WildMenu", { "fg": s:black, "bg": s:blue }) " current match in 'wildmenu' completion
+if get(g:, 'onedark_hide_endofbuffer', 1)
+    call s:h("EndOfBuffer", { "fg": s:black, "bg": s:black }) " Remove '~' at end of buffer for cleaner look. Opt-in in vimrc
```

Comment:
```suggestion
    call s:h("EndOfBuffer", { "fg": s:black, "bg": s:black }) " filler lines (~) after the last line in the buffer.
```

Reply:
The description I have on each highlight line comes straight from `:h highlight-groups`, so I've updated this comment accordingly and moved the explanatory comment above.